projects/manage-supervision-and-delius/deploy/values-preprod.yml

```diff
@@ -1,4 +1,4 @@
-enabled: false # TODO set this to true when you're ready to deploy your service
+enabled: true
```

Comment:
should be able to remove this line.  the default value is set to true.

Reply:
Yep! ive changed both the dev and preprod one